episodes/5-transfer-learning.Rmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Transfer learning"
+teaching: 20
+exercises: 30
+---
+
+::: questions
+- How do I apply a pre-trained model to my data?
+:::
+
+::: objectives
+- Apply a state-of-the-art pre-trained network on your own dataset
+:::
+
+## What is transfer learning?
+Instead of training a model from scratch, with transfer learning you make use of models that are trained on another machine learning task. The pre-trained network captures generic knowledge about the domain during pre-training and will only be 'fine-tuned' to the specifics of your dataset.
```

Comment:
I would remove the 'domain' reference for its ambiguity; it could refer to both the generic domain but also to the domain of your specific dataset.

```suggestion
Instead of training a model from scratch, with transfer learning you make use of models that are trained on another machine learning task. The pre-trained network captures generic knowledge during pre-training and will only be 'fine-tuned' to the specifics of your dataset.
```

Reply:
Good point! Totally agree.